daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/ConcurrentCompiledPackages.scala

```diff
@@ -23,16 +23,20 @@ final class ConcurrentCompiledPackages extends MutableCompiledPackages {
   private[this] val _packageDeps: ConcurrentHashMap[PackageId, Set[PackageId]] =
     new ConcurrentHashMap()
   private[this] var _profilingMode: speedy.Compiler.ProfilingMode = speedy.Compiler.NoProfile
+  private[this] var _stackTraceMode: speedy.Compiler.StackTraceMode = speedy.Compiler.FullStackTrace
 
   def getPackage(pId: PackageId): Option[Package] = _packages.get(pId)
   def getDefinition(dref: speedy.SExpr.SDefinitionRef): Option[speedy.SExpr] =
     Option(_defns.get(dref))
 
-  def stackTraceMode = speedy.Compiler.FullStackTrace
   def profilingMode = _profilingMode
   def profilingMode_=(profilingMode: speedy.Compiler.ProfilingMode) = {
     _profilingMode = profilingMode
   }
+  def stackTraceMode = _stackTraceMode
+  def stackTraceMode_=(stackTraceMode: speedy.Compiler.StackTraceMode) = {
+    _stackTraceMode = stackTraceMode
+  }
```

Comment:
Why do we use explicit trivial getters and setters instead of a public `var`? Scala should generate them for us.

Reply:
`stackTraceMode` is a method from the `CompiledPackages` trait this class implements. It's very unfortunate that Scala's syntax does not make it explicit that I'm overriding a method here. 😦

Reply:
Makes sense! You can add `override` as prefix to `def` to make this clear. (And if there isn't a good reason not to, please do.)

Reply:
Oh, it looks like you can just do:

```
  override var stackTraceMode = speedy.Compiler.FullStackTrace
```

This will add the setter and override the getter.

Reply:
The latter fails to compile with
```
error: variable stackTraceMode overrides nothing
  override var stackTraceMode = _stackTraceMode
```
Changing the `def` to `override def` works though. I'll go with this.